lang/en/block_course_contents.php

```diff
@@ -46,4 +46,10 @@
 $string['config_enumerate_optional_on'] = 'Optional, enabled by default';
 $string['course_contents:addinstance'] = 'Add a new course contents block';
 $string['notusingsections'] = 'This course format does not use sections.';
-$string['pluginname'] = 'Course contents';
\ No newline at end of file
+$string['pluginname'] = 'Course contents';
+$string['config_enumerate_section_0'] = 'Enumerate section 0';
```

Comment:
This could be reworded into more human language. Something like "Include the general section" or "Start enumerating with the general section" or so.